2021-AUG/src/TeamCard.tsx

```diff
@@ -36,14 +36,15 @@ const TeamCard = (props: Props) => {
       <CardActionArea>
         <CardMedia
           className={classes.media}
+          style={{ backgroundSize: "contain" }}
           image={`images/logos/${logo}`}
           title={name}
         />
         <CardContent>
           <Typography gutterBottom variant="h5" component="h2">
             {name}
           </Typography>
-          <Box maxHeight="56px" textOverflow="ellipsis" overflow="hidden">
+          <Box maxHeight="58px" textOverflow="ellipsis" overflow="scroll">
```

Comment:
```suggestion
          <Box maxHeight="58px" textOverflow="ellipsis" overflow="auto">
```

would remove the scrollbar from the bottom of the bio